FCCeeAnalyses/ZH_Zee/dataframe/finalSel.py

```diff
@@ -1,11 +1,14 @@
+from common_defaults import deffccdicts
+
 #python FCCeeAnalyses/ZH_Zmumu/dataframe/finalSel.py 
+import os
 import ROOT
 
 ###Input directory where the files produced at the pre-selection level are
-baseDir  = "/afs/cern.ch/user/h/helsens/FCCsoft/FCCAnalyses/Outputs/FCCee/ZH_Zmumu/"
+baseDir  = "/home/ganis/local/fcc/work/tutorials/aug2020/ch/FCCAnalyses/Outputs/FCCee/ZH_Zmumu/"
```

Comment:
This seems like a local directory, should it also be `FCCDICTSDIR`?

Reply:
Good catch. This is the output dir, cannot be FCCDICTSDIR, must be writable ...
Perhaps we can have FCCANAOUT with default $PWD ?